model/authorizations/v1/resource_review_request_type.model

```diff
@@ -14,6 +14,41 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+enum SubscriptionStatus {
+	// Active state subscriptions have their related resources currently running and reporting an active state
+	// Whether a subscription is active is determined depending on the plan of the subscription
+	// For example, OCP subscriptions are active if the OCP cluster is successfully reporting metrics
+	//              RHOSR subscriptions are active if the service-registry service determines they are, the service updates the subscription as necessary
```

Comment:
NP: extra spaces before 'RHOSR`

Reply:
this is made intentionally to have RHOSR under OCP

Reply:
The maybe it's worth replacing "For example," with "For example: 1) ... 2) ...